source/hardware/controllers/reloop_digital_jockey_2_interface_edition.rst

```diff
@@ -54,7 +54,7 @@ Jog Wheels
 Jog modes work as described in the following table:
 
 .. table::
-    :widths: 70
+    :widths: 70 30
```

Comment:
Thanks for taking care!

Though, let's make it
```suggestion
    :widths: 30 70
```
; )
![image](https://github.com/user-attachments/assets/83fedeb1-0904-4c3f-a208-7c719ba67d76)

Reply:
oh, yeah. Didn't bother checking.